lsp-protocol.el

```diff
@@ -290,6 +290,8 @@ See `-let' for a description of the destructuring mechanism."
 
 ;; begin autogenerated code
 
+(defvar lsp/completion-item-kind-lookup
```

Comment:
How should we use this look up?

Reply:
I'm not sure too how we should use it.
@yyoncho can you give an example using these enums?

Reply:
```
(defcustom lsp-imenu-index-symbol-kinds '(Function Class Interface...))
```

Then, when you receive kind from the language server it can be converted to `Funtion`, `Class`, etc using the lookups autogenerated in this PR.